Description: ensure that resolve() clears computed properties
Author: Ben Burton <bab@debian.org>
Origin: upstream
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- regina-normal-7.3/engine/link/link.cpp
+++ regina-normal-7.3/engine/link/link.cpp
@@ -640,6 +640,8 @@
         crossings_.erase(crossings_.begin() + c->index());
         delete c;
     }
+
+    clearAllProperties();
 }
 
 GroupPresentation Link::group(bool simplify) const {
